tasmota/xdrv_82_esp32_ethernet.ino

```diff
@@ -159,15 +168,28 @@ String EthernetMacAddress(void) {
  * Commands
 \*********************************************************************************************/
 
-#define D_CMND_ETHADDRESS "EthAddress"
-#define D_CMND_ETHTYPE "EthType"
+#define D_CMND_ETHADDRESS   "EthAddress"
+#define D_CMND_ETHTYPE      "EthType"
 #define D_CMND_ETHCLOCKMODE "EthClockMode"
+#define D_CMND_ETHIPADDRESS "Eth" D_CMND_IPADDRESS
+#define D_CMND_ETHGATEWAY   "Eth" D_JSON_GATEWAY
+#define D_CMND_ETHNETMASK   "Eth" D_JSON_SUBNETMASK
+#define D_CMND_ETHDNS       "Eth" D_JSON_DNSSERVER
 
 const char kEthernetCommands[] PROGMEM = "|"  // No prefix
```

Comment:
You can actually use "Eth" as prefix before the first "|"

Reply:
I'll do in a minute
Is there also an way got extract a sub command to avoid the array below?